### PR TITLE
[FEAT] World UX

### PR DESCRIPTION
**Pull request by c-py**
# Description

Adds the World Travel button to all screens and adds some tutorials around the change

## Features

**Travel Tutorial:**
tutorial

**Mail Popup on Login:**
popup

**Labels on Travel Screen:**
labels

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:

- [X] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [ ] Screenshot if it includes any UI changes
- [ ] I have read the contributing guidelines and agree to the T&Cs
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/lib/npcs.ts [hunk 1]
```diff
@@ -469,6 +469,7 @@ export const NPC_WEARABLES: Record<NPCName, Equipped> = {
   // Placeholder fisherman
   finn: {
     body: "Light Brown Farmer Potion",
+    shirt: "Club Polo",
```
Comment (adamhannigan) [inline src/lib/npcs.ts:472]: Ol mate Finn heading to the club 😉

File: src/features/island/hud/components/deliveries/TravelModal.tsx [hunk 2]
```diff
@@ -32,44 +48,74 @@ export const TravelModal: React.FC<Props> = ({ isOpen, onClose }) => {
...
-            { icon: SUNNYSIDE.icons.expression_chat, name: "Chores" },
-          ]}
-          currentTab={tab}
-          setCurrentTab={setTab}
-        >
-          <div
-            style={{ maxHeight: CONTENT_HEIGHT }}
-            className="w-full pr-1 pt-2.5 overflow-y-auto scrollable"
+        {showIntro ? (
+          <Panel bumpkinParts={NPC_WEARABLES["reelin roy"]}>
```
Comment (adamhannigan) [inline src/features/island/hud/components/deliveries/TravelModal.tsx:52]: Any thoughts on some Sailor NPC we could use on this modal?

Do you think there are any wearables which would give the impression of a 'traveller' or 'explorer'

File: src/features/island/hud/components/deliveries/TravelModal.tsx [hunk 3]
```diff
@@ -32,44 +48,74 @@ export const TravelModal: React.FC<Props> = ({ isOpen, onClose }) => {
...
+            <SpeakingText
+              message={[
+                {
+                  text: "Howdy Traveller, ready to set sail?",
```
Comment (adamhannigan) [inline src/features/island/hud/components/deliveries/TravelModal.tsx:56]: It felt like it was missing a bit of context before jumping into the Pumpkin Plaza.

We will add some level gating which means they can only initially visit the Plaza (and won't have other choices). The labels felt very clear to me, so I don't think we need to describe what is at the plaza or beach.

I would suggest the following:

`Howdy Traveller, ready to set sail?`

`Sunflower Land is filled with exciting islands where you can complete deliveries, craft rare NFTs and even dig for treasure!`

`Different locations bring different opportunities to spend your hard earned resources. At any time click the travel button to return home.`

File: src/features/island/hud/WorldHud.tsx [hunk 4]
```diff
@@ -98,7 +100,11 @@ const HudComponent: React.FC = () => {
             }}
           >
             <Leaderboard farmId={farmId} />
-            <DeliveryButton />
+            {hasFeatureAccess(gameState.context.state.inventory, "FISHING") ? (
+              <TravelButton />
```
Comment (adamhannigan) [inline src/features/island/hud/WorldHud.tsx:104]: Fast Travel in the plaza is so good! 💪 Nice idea

**Review by adamhannigan — APPROVED** [review]
Great job! Just some minor suggestions

File: src/assets/announcements/travel_announcement.png
Comment (adamhannigan) [inline src/assets/announcements/travel_announcement.png:1]: Missed this: Our announcement images live in the `images` repo under announcements 👍

**Comment by adamhannigan** [comment]
Screenshot 2023-10-24 at 3 02 37 pm

One small thing, can we show the "!" here as well when there is a new delivery?

The World Icon has a "!" for new deliveries, but I think we should also indicate which tab has the notification

File: src/assets/announcements/travel_announcement.png
Comment (c-py) [reply src/assets/announcements/travel_announcement.png:1]: Ah thanks @adamhannigan !

**Pull request merged**